kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -319,15 +295,11 @@ define([
                     }
                     // treat messages relating to single jobs as if they were for a job list
                     // eslint-disable-next-line no-case-declarations
-                    const jobIdList = msgData[JOB_ID] ? [msgData[JOB_ID]] : msgData[JOB_ID_LIST];
-                    if (msgData.source === REQUESTS.LOGS) {
-                        msgTypeToSend = RESPONSES.LOGS;
-                    } else {
```

Comment:
These log error messages (generally the "can't find a log for job ID...") have been converted in to messages of type `RESPONSES.LOGS` with the error message as part of the content ==> remove this workaround.